core/controllers/suggestion_test.py

```diff
@@ -244,12 +290,12 @@ def test_accept_suggestion(self):
             }, csrf_token=csrf_token, expected_status_int=401)
         self.logout()
 
-        # Testing that author cannot accept own suggestion.
-        self.login(self.AUTHOR_EMAIL_2)
+        # Testing that user cannot accept own suggestion.
```

Comment:
I think author sounds better. The "author" of the suggestion should not be able to perform actions on their own suggestions.

Reply:
Done!